docs/managing-program-users.md

```diff
@@ -3,4 +3,14 @@ id: managing-program-users
 title: Managing Program Users
 ---
 
-Placeholder
+If you are a Program Administrator, then you will have the ability to invite collaborators to access your programs data.
+
+## How to Add Users
+1. Navigate to the **Submission** area in the top menu.
+1. Click **Manage Programs** in the .
```

Comment:
in the left sidebar menu.

Reply:
are you ok with left menu? less words = less to read which usually results in documentation being read more :p

Reply:
sure..or "Programs menu"?